test/functional/interface_bitcoin_cli.py

```diff
@@ -16,7 +16,7 @@ def run_test(self):
         """Main test logic"""
 
         cli_response = self.nodes[0].cli("-version").send_cli()
-        assert "Bitcoin Core RPC client version" in cli_response
+        assert "{} RPC client version".format(self.config['environment']['PACKAGE_NAME']) in cli_response
```

Comment:
Can this be replaced with an `assert_equal` test? That makes it easier to debug whether there's a problem with the testsuite or the binary in AppVeyor.

Reply:
Note that in other places `self` is left out, e.g. `'/src/bitcoind' + config["environment"]["EXEEXT"]`

Reply:
assert_equal doesn't work because the version is suffixed by the commit

```
$ ./src/bitcoin-cli -version
Bitcoin Core RPC client version v0.18.99.0-b1e013e4fa

Reply:
I guess you could strip the commit, but :man_shrugging: